xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-templates/src/main/resources/templates/job/question/ExtensionBreakingQuestion.form.vm

```diff
@@ -17,15 +17,16 @@
 ## Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
 ## 02110-1301 USA, or see the FSF site: http://www.fsf.org.
 ## ---------------------------------------------------------------------------
+##!source.syntax=xwiki/2.1
+{{velocity}}
 #template('job/question/macros.vm')
```

Comment:
If you move to wiki syntax, you should probably move to `{{template name="job/question/macros.vm" output="false"/}}` (but outside the Velocity macro IMO).

Reply:
Not an object anymore since I reverted those changes in regards to https://github.com/xwiki/xwiki-platform/pull/3555#discussion_r1866159228 

$${\color{lightgreen}ADDRESSED}$$